lib/services/static.js

```diff
@@ -11,7 +11,109 @@ var invariant = require('../../vendor/invariant'),
 var MapboxStatic = makeService('MapboxStatic');
 
 /**
- * Determine a URL for a static map image, using the [Mapbox Static Map API](https://www.mapbox.com/developers/api/static/).
+ * Determine a URL for a static map image, using the [Mapbox Static Map API](https://www.mapbox.com/api-documentation/#static).
+ *
+ * @param {string} username
```

Comment:
JSDoc is erroring on username, styleid, bearing, and pitch for not having descriptions